server/etcdserver/apply_v2.go

```diff
@@ -121,6 +121,9 @@ func (a *applierV2store) Sync(r *RequestV2) Response {
 // applyV2Request interprets r as a call to v2store.X
 // and returns a Response interpreted from v2store.Event
 func (s *EtcdServer) applyV2Request(r *RequestV2, shouldApplyV3 membership.ShouldApplyV3) (resp Response) {
+	if r.Method != "PUT" && (storeMemberAttributeRegexp.MatchString(r.Path) || r.Path == membership.StoreClusterVersionKey()) {
+		s.lg.Panic("detected disallowed v2 WAL for stage --v2-deprecation=write-only", zap.String("method", r.Method))
+	}
```

Comment:
From the description of this PR and understood the mix version of 3.5 and 3.6, this `v2PutMemberAttribute` is required to publish 3.5 server readiness, the following condition makes more sense, wdyt? 

```suggestion
	v2PutMemberAttribute = r.Method == "PUT" && (storeMemberAttributeRegexp.MatchString(r.Path) || r.Path == membership.StoreClusterVersionKey()) 
	if !v2PutMemberAttribute {
		s.lg.Panic("detected disallowed v2 WAL for stage --v2-deprecation=write-only", zap.String("method", r.Method))
	}
```

Reply:
Right, make mistake while spitting the PR.